docroot/themes/humsci/su_humsci_theme/theme-settings.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * Provides an additional config form for theme settings.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_system_theme_settings_alter().
+ *
+ * Form override for theme settings.
+ */
+function su_humsci_theme_form_system_theme_settings_alter(array &$form, FormStateInterface $form_state) {
+  $form['humsci_settings'] = [
+    '#type' => 'details',
+    '#title' => t('Humsci Settings'),
+    '#open' => TRUE,
+  ];
+  $form['humsci_settings']['humsci_site_styles'] = [
+    '#type' => 'select',
+    '#title' => t('Choose a site style'),
+    '#empty_option' => t('- Default -'),
+    '#default_value' => theme_get_setting('humsci_site_styles'),
```

Comment:
I've had some fun in the past where `theme_get_setting` wasn't getting the right setting due to theme overrides somewhere, like in subsite.
Since then I have been in the practice of passing in the theme name to that function
Not sure if that will still be a thing in D8